permissions/plugin-aws-sqs.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "aws-sqs-plugin"
+paths:
+- "org/jenkins-ci/plugins/aws-sqs-plugin"
```

Comment:
Thanks for the suggestion. I removed `-plugin` from the `plugin-aws-sqs.yml` and `pom.xml` files.